src/Config.js

```diff
@@ -236,6 +236,10 @@ export class Config {
     return this.customPages.passwordResetSuccess || `${this.publicServerURL}/apps/password_reset_success.html`;
   }
 
+  get parseFrameURL() {
+    return this.customPages.parseFrameURL || null;
```

Comment:
do not use null, we use undefined as bottom value. You should probably check if this.customPages is not undefined.